lib/datadog/appsec/processor.rb

```diff
@@ -103,8 +103,14 @@ def load_ruleset
 
         begin
           @ruleset = case ruleset_setting
-                     when :recommended, :risky, :strict
+                     when :recommended, :strict
                        JSON.parse(Datadog::AppSec::Assets.waf_rules(ruleset_setting))
+                     when :risky
+                       JSON.parse(Datadog::AppSec::Assets.waf_rules(:recommended))
+                       Datadog.logger.warn(
+                         'Ignoring user-supplied ruleset `:risky`. That configuration value has been deprecated,'\
+                         'selecting `:recommended`.'
+                       )
```

Comment:
Putting my user hat on, I think this message could be improved.

---

Imagine working on a rails app with a dozen other people. When you boot the app, there's a bunch of output scrolling: puma this, database that.

And then in between is this message:

> WARN -- ddtrace: [ddtrace] Ignoring user-supplied ruleset `:risky`. That configuration value has been deprecated,selecting `:recommended`.

Would you know what to do, other than go read the ddtrace sources? Or maybe grepping by `risky`?

Here's a suggestion:

> The `:risky` Application Security Management ruleset has been deprecated and no longer available. The `:recommended` ruleset will be used instead. Please remove the `appsec.ruleset = :risky` setting from your `Datadog.configure` block.

I think this message improves on the current one by:
1. Identifying which component this is coming from. Helps googling, talking to support, etc.
2. Documenting the next steps that customers should take to make this warning go away.

Reply:
> has been deprecated

I feel this is incorrect, as usually "deprecation" would mean that it went through a grace period where it was still available.

An option would be to actually deprecate `risky` while keeping things at 1.4.2 for at least one ddtrace release. before we remove `risky` and move to `1.5.0` in a subsequent version.

This way there is no risk of breakage and users of `risky` can move towards either `recommended` or `strict` at their own discretion with due warning.